data/food/soup.json

```diff
@@ -0,0 +1,54 @@
+{
+    "templates": [
+        "hands {user} a {soup} soup."
+    ],
+    "parts": {
+        "soup": [
+" Bowl of Ajiaco ",
```

Comment:
The leading space in this string will result in an output of `hands <user> a  Bowl of Ajiaco ` (note the double space before Bowl and the trailing whitespace). I'd recommend removing surrounding whitespace from these strings.
I'd also recommend running the `format_json.py` script from the repository root to reformat the JSON file to follow the standard format for this repository.